.github/CONTRIBUTING.md

```diff
@@ -260,6 +265,72 @@ existing resources, but you still get to implement something completely new.
    folder. This is to avoid conflicts as the vendor versions tend to be fast
    moving targets.
 
+#### New and Updated Resource Guidelines
+
+The Terraform AWS Provider follows common practices to ensure consistent and reliable implementations across all resources in the project. While there may be older schema and resource code present that predates these guidelines, new submissions are generally expected to adhere to these items to maintain Terraform Provider quality. For any guidelines listed, contributors are encouraged to ask any questions and community reviewers are encouraged to provide review suggestions based on these guidelines to speed up the review and merge process.
+
+The below are required items that will be noted during submission review and prevent immediate merging:
```

Comment:
Does "prevent immediate merging" here mean that immediate merging will be prevented *if* any of these required items are missing? That's been my assumption but the phrasing is a bit ambiguous.

Reply:
Words are hard! What we are trying to relay to contributors here is that these items, if applicable, would be required feedback. We occasionally receive contributions where folks are curious why it is not merged yet and this list would be a reference for those situations. If you have better suggestions for the wording, we can definitely change this. 😄

Reply:
Perhaps:
> the following items, if applicable, are required for new/updates to existing resources?